client/components/Card/Card.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import { Card as Component } from './Card';
```

Comment:
Why did you rename is to `Component`?

Reply:
Because that is what we do on some tests in Leap2. I'm assuming it so that it's easier to replicate tests for other components without having to rename all the tests.